piwik-sdk/src/test/java/org/piwik/sdk/TrackerTest.java

```diff
@@ -248,7 +227,7 @@ public void testGetResolution() throws Exception {
         Tracker tracker = createTracker();
         TrackMe trackMe = new TrackMe();
         tracker.track(trackMe);
-        assertTrue(trackMe.build().contains("res=480x800"));
+        assertTrue(trackMe.build(), trackMe.build().contains("res=480x800"));
```

Comment:
but afaik not because of that change, the first parameter of assertTrue in this case is an error message, i had the same error and wanted to debug it.